mathics/core/definitions.py

```diff
@@ -81,11 +81,13 @@ def __init__(
                             module, remove_on_quit=False
                         )
                     except PyMathicsLoadException as e:
-                        print(e.module + " is not a valid pymathics module.")
-                        continue
+                        raise
```

Comment:
I imagine this was changed for debugging (?). Anyway, we need to clean that too before merging.

Reply:
Because it is better to catch it in the apply method of LoadModule, in order to be able of sending an error message using the method in evaluation.